ros2/launch/launch/launch_example.launch.py

```diff
@@ -26,19 +28,23 @@ def generate_launch_description():
         namespace="core",
         package="params_pkg",
         executable="robot_node",
-        parameters=[{"robot_name":"RobotA", 
-                     "max_speed":4.2, 
-                     "waypoints":["Home", "Room 1", "Corridor", "Home"]}]
+        parameters=[{"robot_name": "RobotA",
+                     "max_speed": 4.2,
+                     "waypoints": ["Home", "Room 1", "Corridor", "Home"]}]
     )
 
     # Launch Foxglove Studio to monitor data
     foxglove_studio = ExecuteProcess(cmd=["foxglove-studio"])
```

Comment:
@defunctzombie is our app binary still called `foxglove-studio` on linux?

Reply:
Yes